board/safety/safety_chrysler.h

```diff
@@ -125,12 +125,21 @@ static int chrysler_tx_hook(CAN_FIFOMailBox_TypeDef *to_send) {
   return true;
 }
 
+static int chrysler_fwd_hook(int bus_num, CAN_FIFOMailBox_TypeDef *to_fwd) {
+  int32_t addr = to_fwd->RIR >> 21;
+  // forward CAN 0 -> 2 so stock LKAS camera sees messages
+  if (bus_num == 0 && addr != 0x2d9 && addr != 0x2a6 && addr != 0x292) {
+    return 2;
+  }
+  return -1;  // do not forward
+}
+
```

Comment:
This is good. there should be a detection of camera messages on BUS 0 though, in order to avoid forwarding when the giraffe is set to "stock"

Reply:
Implemented and tested that it works in a car.